test/integration/user_can_only_modify_own_account_info_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.